specs/wp-manage-domains-spec.js

```diff
@@ -283,4 +291,189 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
 			return await TransferDomainPrecheckPage.Expect( driver );
 		} );
 	} );
+
+	describe.only( 'Register (and cancel) .live domain to an existing site ', function() {
```

Comment:
This `.only` is intentional, so we can run only this test with default test user.